Dockerfile.devkit

```diff
@@ -0,0 +1,37 @@
+FROM debian:buster
+
+ENV LANG=C.UTF-8
+
+RUN set -x \
+    && apt-get update \
+    && apt-get install -y --no-install-recommends \
+        ca-certificates \
+        docker.io \
+        gcc \
+        git \
+        jq \
+        libcurl4-openssl-dev \
+        libssl-dev \
+        python3-dev \
+        python3-pip \
+        python3-setuptools
+
+COPY requirements-dev.txt /marathon-lb/requirements-dev.txt
+COPY requirements.txt /marathon-lb/requirements.txt
+
+# NOTE(jkoelker) dcos-e2e has a large list of strict requrements (== vs >=)
```

Comment:
@jkoelker Please could you make an issue with the epic https://jira.mesosphere.com/browse/DCOS-21525 ("DC/OS E2E Features and Improvements") which describes what you would like the requirements to look like.

Reply:
Doneskis: https://jira.mesosphere.com/browse/DCOS-46542